examples/website-demos/virtual-event/src/components/Chat/ThreadInputUI.tsx

```diff
@@ -0,0 +1,98 @@
+import React, { useCallback } from 'react';
+import { ChatAutoComplete, EmojiPicker, useMessageInputContext } from 'stream-chat-react';
+import { EmojiPickerIcon, GiphyIcon, GiphySearch, SendArrow } from '../../assets';
+import { useGiphyContext } from '../../contexts/GiphyContext';
+
+type Props = {
+  checked: boolean;
+  setChecked: React.Dispatch<React.SetStateAction<boolean>>;
+};
+
+export const ThreadInputUI: React.FC<Props> = (props) => {
```

Comment:
doubt we need to recreate this whole component, can you just add optional props to the standard one and conditionally render the footer?